splunk_eventgen/lib/plugins/generator/jinja.py

```diff
@@ -1,6 +1,8 @@
 from __future__ import division
 from generatorplugin import GeneratorPlugin
-import datetime, time, os
+import datetime
```

Comment:
To keep a good coding style, A better practice is to load system modules before user project modules.
And put a separation line between system modules and project modules.